core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -248,16 +248,16 @@ private void runThreadLocalContextCustomizers() {
     }
 
     private void failEarly(Throwable cause) {
-        final RequestLogBuilder logBuilder = logBuilder();
         final UnprocessedRequestException wrapped = new UnprocessedRequestException(cause);
-        logBuilder.endRequest(wrapped);
-        logBuilder.endResponse(wrapped);
-
         final HttpRequest req = request();
         if (req != null) {
             autoFillSchemeAndAuthority();
-            req.abort(cause);
+            req.abort(wrapped);
```

Comment:
Not sure why it was `cause` before. 🤔

Reply:
It seems my mistake. 😱

Reply:
I think it didn't do any harm anyways. 😆 
First appeared at: https://github.com/line/armeria/pull/2248/files#diff-d437ddb5e0b4572dba776949bacf9becR233